models/api.models.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query("SELECT * FROM topics").then(({ rows }) => {
+    if (rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "No topics were found" });
```

Comment:
We don't need this promise.reject here. We know that the initial data is being seeded so there will be topics in the database. There's no functionality at the moment to delete any topics, so we can assume the rows.length will always be over 0.